public/css/app.css

```diff
@@ -2119,11 +2119,11 @@ select.form-control[multiple], select.form-control[size], textarea.form-control
     padding: .375rem .75rem;
     font-size: .9rem;
     line-height: 1.6;
-    border-radius: .25rem;
+    border-radius: .255rem;
     transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out
 }
```

Comment:
Please order CSS code from A to Z as noted in the Development Rules.

border-radius: .255rem;
font-size: .9rem;
line-height: 1.6;
...

Reply:
this is also code generated